setup.py

```diff
@@ -10,7 +10,8 @@
     version=__version__,
     author='halprin',
     author_email='me@halprin.io',
-    description='Helps spread the AwesomeSauce of the Foundational Components team a bit further',
+    description='Helps spread the AwesomeSauce of the Foundational Components '
+                + 'team a bit further',
```

Comment:
Normally, I wouldn't care too much, but please revert this along with the removing of the unused imports.